website/examples/e-commerce/widgets/PriceSlider.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { connectRange } from 'react-instantsearch-dom';
+
+const RangeWithTooltip = createSliderWithTooltip(Range);
+
+const PriceSlider = ({ min, max, refine, currentRefinement, canRefine }) => {
```

Comment:
The component does not work properly with the min & max value. The slider does not stick to the min & max when those values gets lower and bigger again.  Here is an example:

![Untitled](https://user-images.githubusercontent.com/6513513/59196902-3dce6a00-8b90-11e9-82b6-68cc22a95a09.gif)

Reply:
Indeed, that comes from `rc-slider` triggering the [`onChange`](https://github.com/react-component/slider/#common-api) function whenever a value changes, even without being dragged. That's problematic.